src/Entity/Geo/Zone.php

```diff
@@ -42,6 +43,45 @@ class Zone implements GeoInterface
         self::REGION,
     ];
 
+    public const TYPES_BY_MANDATE = [
```

Comment:
what about moving this into Mandate class instead of keeping it in generic Zone class?

Reply:
done 60510da008b453f73c17bb7c3b18e8577d48e4e8